popSimManu.tex

```diff
@@ -766,9 +772,7 @@ \subsection*{Workflow for analysis of simulated data}
 and specified taking 20 samples each from the Europe and Africa populations.
 For DroMel, runtimes under OutOfAfrica\_2L06 were prohibitively slow when simulating whole genomes.
 % RNG: Removed technical term "coalesce" here, since I don't think that's exactly the issue.
-For example, both chromosomes 3R and X failed to finish within 30 days of starting using $n = 50$.
-Runtimes using the identical sample size were 19, 21, and 12 days for chromosomes 2L, 2R, and 3L,
-respectively. For this reason, we chose to present only data from chromosome 2R for simulations
+For this reason, we chose to present only data from 3 Mb of chromosome 2R for simulations
 under OutOfAfrica\_2L06.
 % RNG: Not present in manuscript
 % For the generic IM simulations, we used the HomSap genome along with the
```

Comment:
just realized that this got commented out - up going to update the PR to uncomment it since we have these results now